app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -37,4 +38,27 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
       ]
     ),
   ])
+  button_group('middleware_server_operations', [
+    select(
+      :middleware_server_power_choice,
+      'fa fa-power-off fa-lg',
+      t = N_('Power'),
+      t,
```

Comment:
Yes, the icons are hard-coded this way. It would be nice if someone cleaned it up so that only the icon name would be passed and the rest would be added in the partial where the markup is generated.